api/routes/posts.js

```diff
@@ -0,0 +1,120 @@
+const express = require("express");
+const User = require("../models/User");
+const Post = require("../models/post");
+
+const app = express();
+// to get all the posts
+app.get("/posts", (req, res) => {
+  Post.findAll({
+    attributes: ["topic", "discription", "city"],
+    include: [
+      {
+        model: User,
+        as: "user",
+        attributes: ["name", "email"],
+      },
+    ],
+  })
+    .then((data) => {
+      console.log(data);
+      res.send(data);
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
Can you add `response.status `in whole routes`(res.status(200) or  res.sendStatus(401))` so that frontend guys first they can check status then go to next steps